example/project/project-scm-1.rockspec

```diff
@@ -10,7 +10,7 @@ dependencies = {
     'checks == 3.0.1-1',
     'http == 1.1.0-1',
     'cartridge == 2.1.2-1',
-    'metrics ~> 0.3'
+    'metrics == scm-1'
```

Comment:
rework when new version is tagged

Reply:
new version of metrics including summary collector is available, 0.5.0